java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S1210_java.html

```diff
@@ -1,6 +1,6 @@
 <p>According to the Java <code>Comparable.compareTo(T o)</code> documentation:</p>
 <blockquote>
-  <p>It is strongly recommended, but not strictly required that <code>(x.compareTo(y)==0) == (x.equals(y))</code>.</p>
+  <p>It is strongly recommended, but not strictly required that <code>(x.compareTo( y )==0) == (x.equals( y ))</code>.</p>
```

Comment:
I am not really sure spaces are done right here. I'm more a person of `(x.compareTo(y) == 0) == (x.equals(y))`

Reply:
agreed, I'm reverting the the change.